crates/pop-common/src/rust_writer_helpers/tests.rs

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: GPL-3.0
+
+use super::*;
```

Comment:
For consistency with the rest of the codebase, we keep unit tests in the same file as the implementation. You can move this code to `rust_writer_helpers`.